README.md

```diff
@@ -175,8 +189,25 @@ flake8 .
 
 Contact [Kosy](https://github.com/kosyfrances), [Delores](https://github.com/Del-sama) or [Nkoli](https://github.com/Nkoli) for more details.
 
+### Using the API endpoint
+
+- Create a superuser account
+
+  ```sh
+  python manage.py createsuperuser
+  ```
+
+- Send a POST request as a JSON object with the username and password values to this [URL](http://localhost:8000/api/token/)
+
+- Enter your access token value which was generated as part of the response in the request authorization tab in the following format:
+  ```sh
+  Authorization: Bearer <Access_Token_Value>
+  ```
+
 ### License
+
 This project is licensed under the MIT License - see the [LICENSE](/LICENSE) file for more details.
 
 ### Acknowledgements
-Special thanks to [Kosy](https://github.com/kosyfrances) and [Delores](https://github.com/Del-sama) for guiding me through this entire app creation process and being my mentors.
+
+Special thanks to [Kosy](https://github.com/kosyfrances), [Delores](https://github.com/Del-sama) and [Tunde](https://github.com/toystars) for guiding us through the entire app creation process, reviewing our code and unblocking us when we are stuck with an issue, and especially for taking the time out of their busy lives to provide free mentorship to us.
```

Comment:
Is all of the readme not from https://github.com/mentorpaired/core/pull/77/files ? 
How did it become a new change on this PR? Was there some issues with rebase?